### PR TITLE
feat(HOME-3086): Prevent route access if MFA configuration is required

### PR DESCRIPTION
**Pull request by antoinemeyer**
https://trizic.atlassian.net/browse/HOME-3086

@kvhswong shall we add this to all other routes under auth as well?  

Corresponding rest-api change: https://github.com/trizic/platform/pull/1623/files#diff-623df1864bac0979d99512da25a56745R57

### CODE REVIEW

**Comment by kvhswong** [comment]
Yes, might as well for all the other routes under auth.

File: api/common/examples/unauthorized.json [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
```
Comment (antoinemeyer) [inline api/common/examples/unauthorized.json]: We don't return a message on unauthorized

**Comment by antoinemeyer** [comment]
That should cover pretty much all the calls

File: api/common/examples/message.res.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "message":"This is a sample message"
```
Comment (kvhswong) [inline api/common/examples/message.res.json]: Space after colon.

**Pull request merged**